Src/Coravel/Queuing/QueueOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Coravel.Queuing
+{
+    public class QueueOptions
+    {
+        public int? ConsummationDelay { get; set; }
```

Comment:
Could we add an XML comment here "This determines how often (in seconds) the queue host will consume all pending tasks."